aimmo-game/simulation/world_map.py

```diff
@@ -289,6 +290,91 @@ def __iter__(self):
                 for y in range(self.min_y(), self.max_y() + 1))
                 for x in range(self.min_x(), self.max_x() + 1))
 
+    # Serialisation Utilities
+    def get_serialised_south_west_corner(self):
+        """
+        Used in serialising the map size when sent to the front end. Very lightweight as it consists of two integers.
+        :return: A dictionary with two values, x and y coordinates for the bottom left corner of the map.
```

Comment:
should remind somewhere that we are working with east, west, north and south

Reply:
Method name contains it, but will include it in the next commit too.